### PR TITLE
Remove label keys from `lite` detections

### PR DESCRIPTION
**Pull request by ntlind**
## Improvements
- Removed label keys from all functions and tests
- Added a DeprecationWarning to the `...from_valor_dict` inner function
- Turn off some API benchmarks since a) they are failing our GH workflows and b) they aren't offering us useful information given the upcoming API changes

### CODE REVIEW

File: integration_tests/benchmarks/object-detection/benchmark_script.py
Comment (czaloom) [inline integration_tests/benchmarks/object-detection/benchmark_script.py:1]: Could you instead comment this out? I think we will need it once we merge lite into the API

File: lite/valor_lite/detection/computation.py [hunk 1]
```diff
@@ -272,9 +276,9 @@ def compute_metrics(
 
     Returns
     -------
-    tuple[NDArray, NDArray, NDArray NDArray]
+    tuple[NDArray, NDArray, NDArray float]
```
Comment (czaloom) [inline lite/valor_lite/detection/computation.py:279]: typo: missing comma

File: lite/valor_lite/detection/computation.py [hunk 2]
```diff
@@ -272,9 +276,9 @@ def compute_metrics(
...
         Average Precision results.
-    tuple[NDArray, NDArray, NDArray NDArray]
+    tuple[NDArray, NDArray, NDArray float]
```
Comment (czaloom) [inline lite/valor_lite/detection/computation.py:281]: typo: missing comma

File: integration_tests/benchmarks/object-detection/benchmark_script.py
Comment (ntlind) [reply integration_tests/benchmarks/object-detection/benchmark_script.py:1]: sure, the only change to this file is to set the default for compute_pr, etc. to False

File: lite/tests/detection/test_stability.py [hunk 3]
```diff
@@ -17,7 +17,7 @@ def bbox(is_prediction):
             xmax,
             ymin,
             ymax,
-            [("class", choice(labels)), ("category", choice(labels))],
+            [choice(labels), choice(labels)],
```
Comment (czaloom) [inline lite/tests/detection/test_stability.py:20]: This doesnt make sense anymore, there should only be one groundtruth per box

File: lite/tests/detection/test_filtering.py
Comment (czaloom) [inline lite/tests/detection/test_filtering.py:1]: With the modification to the basic detections fixture the label filter tests in this file aren't tested enough. Could you make a fixture with multiple labels and see that the filters perform the same?

File: lite/tests/detection/test_average_precision.py [hunk 4]
```diff
@@ -37,60 +37,52 @@ def test__compute_average_precision():
...
             [1.0],  # iou = 0.1
             [1 / 3],  # iou = 0.6
         ]
     )
-    assert expected_ap.shape == average_precision.shape
-    assert np.isclose(average_precision, expected_ap).all()
+    assert expected.shape == average_precision.shape
+    assert np.isclose(average_precision, expected).all()
 
-    # since only one class, ap == map
```
Comment (czaloom) [inline lite/tests/detection/test_average_precision.py:49]: this comment might be good to retain

**Review by czaloom — APPROVED** [review]

**Pull request merged**